multiqc/utils/search_patterns.yaml

```diff
@@ -272,6 +272,9 @@ deeptools/plotPCAData:
 deeptools/plotProfile:
   contents: "bin labels"
   num_lines: 1
+diamond:
+  contents: "type: Diamond database"
+  fn: "*.log"
```

Comment:
Does this module parse only the output from the  `--log` flag? https://github.com/bbuchfink/diamond/wiki/3.-Command-line-options#general-options

In which case, is the filename _always_ `diamond.log`? That's much easier (and faster) to search for..

Reply:
Talked to @jfy133 and it looks like this is the case. I updated the module accordingly in https://github.com/ewels/MultiQC/commit/a48995423992146f41e1f7be423bec40259db1ab:

- Update search pattern
- Parse sample name from logged command instead of filename
- Fall back to log directory name

Note that if you're manually renaming these log files in the taxprofiler pipeline, you'll need to tweak the search pattern in the pipeline MultiQC config file for it to still find the files. See [docs](https://multiqc.info/docs/#module-search-patterns).